<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="delete_btn">
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="specify_attr_lbl">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>27</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Specify Attr:</string>
   </property>
  </widget>
  <widget class="QLabel" name="attr_type_lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>27</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Attr Type:  </string>
   </property>
  </widget>
  <widget class="QComboBox" name="attr_type_ddp">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>27</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
   <item>
    <property name="text">
     <string>Any</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Float</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Enum</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="specify_attr_checkBx">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>66</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>OR</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
